src/main/java/com/audiolibrary/web/model/Artist.java

```diff
@@ -0,0 +1,55 @@
+package com.audiolibrary.web.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.Set;
+
+
+@Entity
+public class Artist {
+
+    @OneToMany(mappedBy = "artist", fetch = FetchType.EAGER, cascade = CascadeType.REMOVE)
+    @JsonIgnoreProperties("artist")
+    private Set<Album> albums;
```

Comment:
C'est bien de prendre l'habitude d'initialiser les collections pour être tranquille lors de l'ajout d'éléments
```suggestion
    private Set<Album> albums = new HashSet<>();
```

Reply:
Comme c'est un HashSet, c'est important de redéfinir hashCode dans Album